pkg/daemon/ceph/client/mgr.go

```diff
@@ -105,14 +104,13 @@ func MgrDisableModule(context *clusterd.Context, clusterInfo *ClusterInfo, name
 }
 
 // MgrSetConfig applies a setting for a single mgr daemon
-func MgrSetConfig(context *clusterd.Context, clusterInfo *ClusterInfo, mgrName string, key, val string, force bool) (bool, error) {
+func MgrSetConfig(context *clusterd.Context, clusterInfo *ClusterInfo, daemonID string, key, val string, force bool) (bool, error) {
```

Comment:
Seems like this function should be renamed to not mention `Mgr` and possibly be moved out of `mgr.go`?

Reply:
And "failed to set mgr config key" error message could be adjusted.